### ISSUE TITLE
Goto definition (F12) is overriden on Linux

### ISSUE THREAD

**Issue by lzap** [damccull/sublimetext-SolarizedToggle#19]
Hello,

not sure if you use Linux, but default F12 key is bound to "goto_definition". This is quite useful key and I got used to it. Would you mind changing the default key for toggle to Ctrl+F12 on that platform too? Thanks!

**Comment by lzap** [comment]
Workaround for googlers, put this in the users keymap file:

```json
{
    { "keys": ["ctrl+f12"], "command": "solarized_toggle" },
    { "keys": ["f12"], "command": "goto_definition" }
}
```

**Comment by damccull** [comment]
Merged #21.
